modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/client/ItThinClientMarshallingTest.java

```diff
@@ -233,13 +235,19 @@ public void testIncompatiblePojoFieldType2() {
 
     @Test
     public void testIncompatibleTupleElementType() {
-        Table table = ignite().tables().table(TABLE_NAME);
+        var tableName = "testIncompatibleTupleElementType";
+        ignite().sql().execute(null, "CREATE TABLE " + tableName + " (KEY INT PRIMARY KEY, VAL VARCHAR NOT NULL)");
+
+        Table table = ignite().tables().table(tableName);
         var tupleView = table.recordView();
 
-        Tuple rec = Tuple.create().set("KEY", "1").set("VAL", BigDecimal.ONE);
+        Tuple rec = Tuple.create().set("KEY", 1).set("VAL", 1L);
 
+        // The validation done on a client side (for a thin client), and messages may differ between embedded clients and thin clients.
+        // For an embedded client the message include type precision, but for a thin client it doesn't.
```

Comment:
It sounds not so good. Should we fix messages?

Reply:
Seems should be fixed under https://issues.apache.org/jira/browse/IGNITE-22965. Is it so, let's just add mention the ticket here

Reply:
Done.